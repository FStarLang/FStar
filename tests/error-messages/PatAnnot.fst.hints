[
  "%×õ¹£™E\u0016r×\u00052&ç \u000b",
  [
    [
      "PatAnnot.test2",
      1,
      2,
      1,
      [
        "@MaxIFuel_assumption", "@query", "b2t_def", "equation_Prims.squash",
        "primitive_Prims.op_GreaterThanOrEqual",
        "projection_inverse_BoxBool_proj_0",
        "refinement_interpretation_Tm_refine_2de20c066034c13bf76e9c0b94f4806c"
      ],
      0,
      "bd82941eaa93736860ea3bcd70ce92c1"
    ],
    [
      "PatAnnot.test4",
      1,
      2,
      1,
      [
        "@MaxIFuel_assumption", "@query", "equation_Prims.nat",
        "lemma_FStar.Pervasives.invertOption",
        "refinement_kinding_Tm_refine_542f9d4f129664613f2483a6c88bc7c2"
      ],
      0,
      "dbc0878b1f4a542f6a56cd25968da57b"
    ]
  ]
]