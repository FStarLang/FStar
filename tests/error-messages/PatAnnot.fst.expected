>> Got issues: [
PatAnnot.fst(25,8-25,9): (Error 19) Subtyping check failed; expected type Prims.squash Prims.l_False; got type Prims.unit; The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel (see also PatAnnot.fst(25,19-25,24))
>>]
>> Got issues: [
PatAnnot.fst(28,0-30,14): (Error 19) assertion failed (Also see: prims.fst(429,55-429,67)); The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel
>>]
>> Got issues: [
PatAnnot.fst(33,0-35,14): (Error 19) assertion failed (Also see: prims.fst(429,55-429,67)); The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel
>>]
>> Got issues: [
PatAnnot.fst(39,10-39,12): (Error 19) Subtyping check failed; expected type Prims.squash Prims.l_False; got type Prims.unit; The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel (see also PatAnnot.fst(40,26-40,31))
>>]
>> Got issues: [
PatAnnot.fst(46,10-46,11): (Error 19) Subtyping check failed; expected type Prims.nat; got type Prims.int; The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel (see also prims.fst(676,18-676,24))
>>]
>> Got issues: [
PatAnnot.fst(55,36-55,39): (Error 19) Type annotation on parameter incompatible with the expected type; The SMT solver could not prove the query, try to spell your proof in more detail or increase fuel/ifuel (see also prims.fst(676,18-676,24))
>>]
Verified module: PatAnnot
All verification conditions discharged successfully
